src/main/java/org/openrewrite/java/spring/doc/MigrateDocketBeanToGroupedOpenApiBean.java

```diff
@@ -0,0 +1,304 @@
+/*
+ * Copyright 2025 the original author or authors.
+ * <p>
+ * Licensed under the Moderne Source Available License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://docs.moderne.io/licensing/moderne-source-available-license
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.spring.doc;
+
+import lombok.Getter;
+import lombok.Value;
+import org.jspecify.annotations.Nullable;
+import org.openrewrite.*;
+import org.openrewrite.java.*;
+import org.openrewrite.java.search.UsesType;
+import org.openrewrite.java.spring.AddSpringProperty;
+import org.openrewrite.java.spring.IsPossibleSpringConfigFile;
+import org.openrewrite.java.tree.Expression;
+import org.openrewrite.java.tree.J;
+import org.openrewrite.marker.SearchResult;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Consumer;
+
+import static org.openrewrite.Preconditions.and;
+import static org.openrewrite.Preconditions.or;
+
+public class MigrateDocketBeanToGroupedOpenApiBean extends ScanningRecipe<MigrateDocketBeanToGroupedOpenApiBean.DocketBeanAccumulator> {
```

Comment:
Which composite recipe should now include `org.openrewrite.java.spring.doc.MigrateDocketBeanToGroupedOpenApiBean` ?
Because we do not want users to have to run this as a one-off recipe separately.

Reply:
I'm guessing it's this one, but you'll want to look a little more closely perhaps:
https://github.com/openrewrite/rewrite-spring/blob/1cd94adaa20767263778ae6984123d229dfc357a/src/main/resources/META-INF/rewrite/springdoc.yml#L75-L81

Reply:
No, you need to add it to the `SwaggerToSpringDoc` recipe:
https://github.com/openrewrite/rewrite-spring/blob/1cd94adaa20767263778ae6984123d229dfc357a/src/main/resources/META-INF/rewrite/springdoc.yml#L34-L41